README.md

```diff
@@ -31,6 +31,11 @@ Instructions can be found here: https://formulae.brew.sh/formula/node@12
 **brew**
 ```bash
 brew install node@12
+
+# Note: After this, you may need to update your PATH. Keep an eye on the output of brew.
+# Example Output:
+# If you need to have node@12 first in your PATH, run:
+#  echo 'export PATH="/usr/local/opt/node@12/bin:$PATH"' >> ~/.zshrc
```

Comment:
Doesn't brew symlink automagically?

Reply:
Mine didn't. I've had the experience where it just doesn't do this if you specify a version that isn't the widely accepted version, but I can confirm after running install, `which npm` and `which node` gave me nothing, but then once I explicitly added to my zshrc (effectively as suggested), that I had access.